definitions.h

```diff
@@ -223,10 +229,26 @@ class BufferImage : public Buffer2D<PIXEL>
 class Attributes
 {      
     public:
+        //color
+        double r;
+        double g;
+        double b;
+        //image
+        double u;
+        double v;
+
```

Comment:
Instead of separate values for rgb and uv, you could implement a double array of size 16. What I did is create a vector that I pushed back values to and looped through later in the interpolation.